### PR TITLE
Added play video icon on ReactPlayer of src/sections/Home/CloudNative…

### LINKED ISSUE #4012: low contrast of play button
### Screenshots / Mockups

Screenshot (100)

Screenshot (99)

@leecalcote  if this is an issue can i work on this?

### PR DESCRIPTION
**Pull request by bhaskarcsawant**
…Management component.

**Description**

This PR fixes #4012

**Notes for Reviewers**
This PR adds a Play icon to the ReactPlayer component

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 
addedPlayIcon

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0389b985282345ac244ab94116cb248f6c110f8f at: https://6423f154af4b8a72ab78086b--layer5.netlify.app

**Review by leecalcote — COMMENTED** [review]
This is looking pretty good, @bhaskarcsawant 👍 

A request: on hover of the screenshot (large thumbnail), will you have the play icon animate in some form (remove drop shadow, or change color, or something similar)?

**Comment by leecalcote** [comment]
Resyncing with `master`

**Comment by l5io** [comment]
🚀 Preview for commit 8f13984efa3b9acb0fa5ddd7d582440f9c9e8b0a at: https://6426ff59a7164000a7fad232--layer5.netlify.app

**Comment by leecalcote** [comment]
> > Resyncing with `master`
> 
> Heyy , the project is not working at all in local ... stucked on image processing job..🤧

That's unfortunate. See https://discuss.layer5.io for threads on this topic and suggestions.

File: src/sections/Home/CloudNativeManagement/index.js [hunk 1]
```diff
@@ -59,6 +60,18 @@ const BannerDefault = () => {
               light={< img className="imgHero" src={imgHero} loading="lazy" alt="meshmap designer" width="100%" height="100%" />}
               width="auto"
               height="100%"
+              playIcon={
+                <img
+                  loading="lazy"
```
Comment (Chadha93) [inline src/sections/Home/CloudNativeManagement/index.js:65]: Add these styles in class.

**Comment by l5io** [comment]
🚀 Preview for commit 4618c37d1eb2021b9b76cd9d099ace3cc3f4d80c at: https://6433facac1587d0de9d1de3f--layer5.netlify.app

**Comment by leecalcote** [comment]
@bhaskarcsawant, if you're having trouble building the layer5.io website, perhaps, you might want to jump on meshery.io or one of the other 8 or so sites/front-end apps - https://layer5.io/community/handbook/repository-overview

**Comment by Nikhil-Ladha** [comment]
@bhaskarcsawant did we loose you here?

**Pull request closed without merging**